src/App.js

```diff
@@ -29,6 +29,7 @@ import {Column} from './common/styles';
 import {Footer} from "./components/footer/Footer";
 
 import './App.css';
+import ShapesAndOntologies from "./components/shapesAndOntologies/ShapesAndOntologies";
```

Comment:
Just a small preference, but please organise imports as following:
1st are the imports regarding external npm packages, default imports come first.
2nd are the imports of our written components or resources.
3nd are the style related imports but this should be only the CSS modules.

Reply:
I updated all imports and add the rules in eslint